docs/application/dotnet/guides/nui/nui-components/Popup.md

```diff
@@ -0,0 +1,61 @@
+# Popup
+This tutorial describes how to create and use Popup.
+
+## Overview
+Popup is one kind of common component, it can be used as popup window.
+User can handle Popup button count, head title and content area.
+
+## Create with property
+1. Create Popup by default constructor
+
+~~~{.cs}
+Popup popup = new Popup();
+~~~
+
+2. Set popup property
+
+~~~{.cs}
+//set Popup property
+NPatchVisual nvisual = new NPatchVisual();
+nvisual.URL = "popup_background.png";
+nvisual.Border = new Rectangle(0, 0, 81, 81);
+popup.MinimumSize = new Size2D(1032, 184);
+popup.Size = new Size(1032, 400);
+popup.Position = new Position(200, 100);
+
+//set Popup title property
+popup.TitlePointSize = 25;
+popup.TitleTextColor = Color.Black;
+popup.TitleHeight = 68;
+popup.TitleTextHorizontalAlignment = HorizontalAlignment.Begin;
+popup.TitleTextPosition = new Position(64, 52);
+popup.TitleText = "Popup Title";
+
+popup.ButtonTextColor = color[index];

```

Comment:
`color[index]` is not part of this code. Please replace it to proper color value.

Reply:
Done